lib/tasks/event.rake

```diff
@@ -79,15 +79,28 @@ namespace :crossref do
 end
 
 namespace :subj_id_check do
-  desc 'checks that events subject node congruency'
-  task :check => :environment do
-    from_id = (ENV['FROM_ID'] || Event.minimum(:id)).to_i
-    until_id = (ENV['UNTIL_ID'] || Event.maximum(:id)).to_i
-    
+  desc "checks that events subject node congruency"
```

Comment:
Can you use a description that is easier to understand?

Reply:
this tasks is for another purpose, i was just fixing linting.

nevertheless i added a better description and moved into another task to get a better context of it use